packages/site-kit/components/Nav.svelte

```diff
@@ -20,8 +20,14 @@
 		open = false;
 	});
 
+	let window_inner_width;
+	function is_mobile() {
+		if (window_inner_width < 768) return true;
+		return false;
+	}
+
 	function intercept_touchstart(event) {
-		if (!open) {
+		if (!open && is_mobile()) {
 			event.preventDefault();
```

Comment:
I wonder if we even need this line and the next. Even after the menu is open or when I'm on my laptop, clicking the link for the current page does nothing

Reply:
I see what you mean - I think this is another issue. The menu is closed when the page store changes (i.e. on navigation). I'm presuming that SvelteKit doesn't change this store if the navigation is to the same page. So this would leave the menu open when clicking the current page's link.

```javascript
	// hide nav whenever we navigate
	page.subscribe(() => {
		open = false;
	});
```

Reply:
It's not really an issue though, is it? The behavior seems fine. So I'm wondering if we can take advantage of this behavior to simplify the code and just remove the `preventDefault` and `stopPropagation`

Reply:
Ok sorry misunderstood. The `preventDefault` is needed to stop the menu navigating to the home page when you touch to open it. This is what was happening in #33. But I can't see a need for `stopPropagation` so that can probably be removed. I will test that now.

Reply:
I still don't understand why `preventDefault` is needed. Let's say you're on `/faq`. Then that's what the menu will say. If you click it then SvelteKit's router will handle the click, see the URL matches the current URL, and do nothing. So it seems to me like you could just let the click event be handled. Am I missing something?

Reply:
I can make a repro later today.

If you're on `/faq` and you touch the dropdown link the menu opens. Because the menu opens on touch though, it opens before the click event fires. Regardless of the current page, it's always the `Home` button that is directly under the cursor when the menu is open. The click event then fires and it navigates to `Home`.

![image](https://user-images.githubusercontent.com/10662340/116946466-87148f00-acbd-11eb-9004-d610aae45313.png)

![image](https://user-images.githubusercontent.com/10662340/116946488-9267ba80-acbd-11eb-982e-ea62a39c1d98.png)

Reply:
Here is the current state of this PR but with the `preventDefault` removed - http://kit-no-preventdefault.surge.sh/. I have put my proposed approach in the _Todo_ section of the PR, but open to any alternative suggestions.

Many thanks for your time reviewing this.